recipes/sos-julia/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sos-julia" %}
+{% set version = "0.18.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 53551d793ef9ac37dc2df39a4978321d3231e119c3a740edd0c924b38f6c9dbc
+
+build:
+  noarch: python
+  number: 0
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
+    - export JUPYTER_DATA_DIR=${PREFIX}/share/jupyter
+    - export JUPYTER=$(which jupyter)
+    - julia -e "using Pkg;Pkg.add([\"IJulia\", \"Feather\", \"DataFrames\", \"NamedArrays\"])"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
```

Comment:
Python should probably be a run requirement too

Reply:
done.